index.html

```diff
@@ -80,6 +80,13 @@ <h2>About Us</h2>
 </section>
 <hr />
 
+<section id="society-rules">
+  <h2>Society Rules</h2>
+
+  <p>Read about the Rules of Ruby New Zealand Incorporated from <a href="society-rules.pdf" target="_blank">here</a>.</p>
```

Comment:
```suggestion
  <p><a href="society-rules.pdf" target="_blank">Read about the Rules of Ruby New Zealand Incorporated from here.</a></p>
```

Reply:
@warp what do you think about this? https://github.com/rubynz/www/pull/68/files#r786407816